imageio_ffmpeg/_definitions.py

```diff
@@ -1,25 +1,27 @@
 import platform
 import struct
-import sys
 
 __version__ = "0.5.0"
 
 
 def get_platform():
     bits = struct.calcsize("P") * 8
-    if sys.platform.startswith("linux"):
+    if platform.system().lower().startswith("linux"):
```

Comment:
It's not strictly necessary to switch from `sys` to `platform` here, but it does feel a little simpler to stick with a single library for the function.